webpack.config.js

```diff
@@ -0,0 +1,10 @@
+const path = require("path");
+
+module.exports = {
+    mode: "production",
+    entry: "./js/app/parser/index.js",
+    output: {
+        filename: "index.built.js",
+        path: path.resolve("./js/app/parser")
```

Comment:
can you use `path.join()` here,  I remember we encountered a similar issue in eslint repo somehow.

Reply:
The `path` field must be absolute path, while using `path.join` won't return that.

Reply:
I think this is fine, though I do think it's common to use `__dirname`. i.e. `path.resolve(__dirname, "js/app/parser")`.